build.gradle

```diff
@@ -733,7 +733,6 @@ task dockerUpload {
   || (dockerBuildVersion ==~ /.*-beta/) || (dockerBuildVersion ==~ /.*-RC/)
```

Comment:
I think this should also be changed to *-RC*

Reply:
+1